guides/common/modules/proc_enabling-the-installer-managed-dhcp-service.adoc

```diff
@@ -0,0 +1,65 @@
+[id="enabling-the-installer-managed-dhcp-service_{context}"]
+= Enabling the installer-managed DHCP service
+
+// MARC: New module. Content is based on https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_reverting-to-internal-dns-service.adoc and https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc, and adjusted for TFTP.
+// Needs to be checked if it is technically correct.
+
+You can revert to using {ProjectServer} and {SmartProxyServer} as your DHCP providers.
+You can use a backup of the answer file that was created before configuring external DHCP, or you can create a backup of the answer file.
```

Comment:
I don't like the idea of "revert". IMHO the path for users should be that they have no DHCP server and then start using the installer managed option. Users should explicitly specify the options.

@ehelms do you agree with me?